src/ContractInteract/OSTPrime.js

```diff
@@ -43,6 +60,121 @@ class OSTPrime {
     this._wrapRawTx = this._wrapRawTx.bind(this);
     this.unwrap = this.unwrap.bind(this);
     this._unwrapRawTx = this._unwrapRawTx.bind(this);
+    this.setCoGateway = this.setCoGateway.bind(this);
+    this.setCoGatewayRawTx = this.setCoGatewayRawTx.bind(this);
+    this.initialize = this.initialize.bind(this);
+    this.initializeRawTx = this.initializeRawTx.bind(this);
+  }
+
+  /**
+   * Setup for OSTPrime contract. Deploys the contract and initializes
+   * it. See {@link OSTPrime#initialize}.
+   *
+   * @param {Web3} web3 Web3 object.
+   * @param {OSTPrimeSetupConfig} config OSTPrime setup configuration.
+   * @param {Object} txOptions Transaction options.
+   *
+   * @param {Promise<OSTPrime>} Promise containing the OSTPrime instance that
+   *                           has been set up.
+   */
+  static setup(web3, config, txOptions) {
+    if (!config.valueToken) {
+      throw new Error(
+        'Mandatory configuration "valueToken" missing. Provide EIP20 token contract address.',
+      );
+    }
+
+    if (!config.organization) {
+      throw new Error(
+        'Mandatory configuration "organization" missing. Set config.organization address.',
+      );
+    }
+
+    OSTPrime.validateSetupConfig(config);
```

Comment:
Why is this different from the two `if`s above?

Reply:
There was a requirement in the old `ChainSetup#setup` for it to be that way, but should be able to be moved to `validateSetupConfig` now.